src/WebJobs.Script.WebHost/IFileMonitoringService.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public interface IFileMonitoringService : IHostedService
```

Comment:
This should also expose an API to resume the restart monitoring, even if that's provided by the disposable result.

Reply:
I do not know how we are going to use the method assuming we can call `Dispose` from the code. Should it just decrease `suspensionRequestCount `or set it to 0?

Reply:
I added `ResumeRestart` method